src/moneywiz_api/model/transaction.py

```diff
@@ -489,11 +505,7 @@ def validate(self):
 
         assert self.original_exchange_rate is not None
 
-        assert self.amount == self.original_amount
```

Comment:
amount and original_amount should always be same, if assertion on lines 477 481 are true.